lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -129,7 +129,14 @@ def stop_repmgrd
     end
 
     def node_number_valid?
-      rec = record_for_node_number
+      begin
+        rec = record_for_node_number
+      rescue PG::Error => e
+        error_msg = "Failed to record for node number #{node_number}. #{e.message}"
```

Comment:
All agree but have doubt on this one. Error can be error on connection or execute the `SELECT ... FROM repl_nodes` in `record_for_node_number`. It's a `PG::UndefinedTable` Error in my vm. Do we still use `Failed to connect`?